modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -58,6 +64,52 @@ public String getFilesystemFriendlyName() {
                 assertEquals("Recorded file not found", 1, files.length);
             }
         }
+
+        @Test
+        public void recordingTestThatShouldHaveCorrectDuration() throws IOException {
+            final String flvFileTitle = "ChromeThatRecordsAllTests-recordingTestThatShouldBeRecordedAndRetained";
+            final String flvFileNameRegEx = "PASSED-" + flvFileTitle + ".*\\.flv";
+
+            try (
+                BrowserWebDriverContainer chrome = new BrowserWebDriverContainer()
+                    .withCapabilities(new ChromeOptions())
+                    .withRecordingMode(RECORD_ALL, vncRecordingDirectory.getRoot())
+                    .withRecordingFileFactory(new DefaultRecordingFileFactory())
+            ) {
+                chrome.start();
+
+                doSimpleExplore(chrome);
+                chrome.afterTest(new TestDescription() {
+                    @Override
+                    public String getTestId() {
+                        return getFilesystemFriendlyName();
+                    }
+
+                    @Override
+                    public String getFilesystemFriendlyName() {
+                        return flvFileTitle;
+                    }
+                }, Optional.empty());
+
+                String recordedFile = vncRecordingDirectory.getRoot().listFiles(new PatternFilenameFilter(flvFileNameRegEx))[0].getCanonicalPath();
+
+                ToStringConsumer dockerLogConsumer = new ToStringConsumer();
+
+                try( GenericContainer container = new GenericContainer<>("jrottenberg/ffmpeg:3.2-alpine38") ) {
+                    container.withStartupCheckStrategy(new OneShotStartupCheckStrategy())
+                            .withFileSystemBind(recordedFile, "/tmp/chromeTestRecord.flv", BindMode.READ_WRITE )
+                            .withCommand("-i" ,"/tmp/chromeTestRecord.flv")
+                            .withLogConsumer(dockerLogConsumer)
+                            .start();
+                } catch (RuntimeException ex) {
```

Comment:
`OneShotStartupCheckStrategy ` should not throw an exception if the container stops with exit code 0.

Reply:
Fixed.